libamqpprox/amqpprox_connector.cpp

```diff
@@ -386,11 +389,19 @@ void Connector::sendResponse(const T &response, bool sendToIngressSide)
 }
 
 template <typename TReply, typename TMethod>
-inline void Connector::synthesizeMessage(TMethod &replyMethod,
-                                         bool     sendToIngressSide)
+inline void
+Connector::synthesizeMessage(TMethod &replyMethod,
+                             bool     sendToIngressSide,
+                             uint64_t code /* = Reply::Codes::reply_success */,
```

Comment:
Can we always use `code` and `text` and remove the templated argument `TReply` here? I don't see the benefit of it